db2-contacts.js

```diff
@@ -4,28 +4,33 @@ const pl = require('pull-level')
 const Plugin = require('ssb-db2/indexes/plugin')
 const isFeed = require('ssb-ref').isFeed
 
-const bValue = Buffer.from('value')
-const bAuthor = Buffer.from('author')
-const bContent = Buffer.from('content')
-const bType = Buffer.from('type')
-const bContact = Buffer.from('contact')
-
-module.exports = function (createLayer) {
+const B_VALUE = Buffer.from('value')
+const B_AUTHOR = Buffer.from('author')
+const B_CONTENT = Buffer.from('content')
+const B_TYPE = Buffer.from('type')
+const B_CONTACT = Buffer.from('contact')
```

Comment:
Using all caps helped to signal that these are constants, and it was getting easy for me to mix up `bContent` and `pContent`, so the all caps helps stand out from the camel case.

Reply:
:camel: approves